src/main/java/kermit/Ui.java

```diff
@@ -0,0 +1,113 @@
+package kermit;
+
+import kermit.tasks.Task;
+
+/**
+ * Ui handles interface with user such as receiving user input and printing messages to user.
+ */
+public class Ui {
+    private static final String introductionText =
+            "Hello I am Kermit ( *・∀・)ノ゛, eaten any flies today?\nWhat can I do for you?";
+    private static final String listText = "Here are the tasks in your list:";
+    private static final String completeTaskText = "Ribbit Ribbit! Good job, task has been marked as complete:";
+    private static final String filteredTaskText = "Here are the matching tasks in your list:";
+    private static final String goodbyeText = "Bye. Hope to see you again soon!";
+    private static final String errorText = "Burp burp! Something went wrong!";
+    private static final String loadingErrorText = "Could not read this file Nuuuuuuu!";
+
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+    }
+
+    /**
```

Comment:
Sorry for commenting on this again! The course required leaving a review comment instead of a normal comment. Still think your code is great but also thought it was impressive how much documentation you provided.